astrosat_users/adapters.py

```diff
@@ -294,6 +294,14 @@ def send_password_confirmation_email(self, user, email, **kwargs):
 
         self.send_mail(template_prefix, email, context)
 
+    def send_mail(self, template_prefix, email, context, **kwargs):
+        cc = kwargs.pop("cc", [])
+        bcc = kwargs.pop("bcc", [])
```

Comment:
More for my learning, but is it possible to write this as

```
        def send_mail(self, template_prefix, email, context, cc=[], bcc=[]):
```

I get that `kwargs` allows you to add more parameters to the function, without having to keep adding to the above list.

Reply:
Yes, I could have written it as you suggest.  In general, I use `**kwargs` (and`*args`) when I don't know the signature of the function I am overwriting or when I think it might change.  In this case, I naively assumed that `allauth` passed `**kwargs` onto the `render_email` function but it turned out I was mistaken.